form.html

```diff
@@ -0,0 +1,141 @@
+        <img src="" alt="Логотип Кэт энерджи." class="main-header__logo-img" >
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню.</span>
+        </button>
+      </nav>
+      <div class="main-nav__wrapper">
+        <ul class="site-list">
+<li class="site-list__item">
+  <a href="#" class="site-list__item-link">
+    Главная
+  </a>
+</li>
+<li class="site-list__item">
+  <a href="#" class="site-list__item-link">
+Каталог продукции
+  </a>
+</li>
+<li class="site-list__item">
+  <a href="#" class="site-list__item-link">
+  Подбор программы
+  </a>
+</li>
+  </ul>
+      </div>
+    </header>
+
+<main class="main-form">
+   <section class="main-form-container">
+    <h1 class="main-form__title">
+        Подбор программы
+    </h1>
+    <p class="main-form_subtitle">
+        Заполните анкету, и мы подберем
+программу питания для вашего кота
+     </p>
+    <form action="index.html"  method="post" class="main-form__form-ance form">
+<label class="form__data" for="name">Имя:<sup>*</sup></label>
+        <input id="name" type="text" name="name" placeholder="БАРСИК">
+        <label class="form__box" for="weight">Вес (кг):<sup>*</sup></label>
+        <input id="weight" type="number" name="weight" placeholder="7">
+        <label class="form__box" for="age">Возраст (лет):</label>
+        <input id="age" type="number" name="age" placeholder="7">
+    </form>
+    <form action="index.html"  method="post" class="main-form__form-radio form">
+        <input id="slim" type="radio" name="goal" value="slim" checked>
+        <label class="form__goal" for="slim">Похудение</label>
+        <input id="fat" type="radio" name="goal" value="fat" >
+        <label class="form__goal" for="fat">Набор массы</label>   
+        <input id="unknown" type="radio" name="goal" value="unknown"  >
+        <label class="form__goal" for="unknown">Не знаю (Нужен ваш совет)</label>    
+            </form>
+            
+            <form action="index.html" class="main-form__contacts form">
+                <fieldset>
+                    <legend>Контактные данные (владельца кота)</legend>
+                <label class="form__box form__box--letter" for="e-mail">E-mail:<sup>*</sup></label>
+                <input id="e-mail" type="email" name="e-mail" placeholder="kuklachev@gmail.com">
+                <label class="form__box form__box--phone" for="phone">Телефон:<sup>*</sup></label>
+                <input id="phone" type="number" minlength="11" name="phone" placeholder="8 (960) 900-60-90">
+                </fieldset>
+            </form>
+            <form action="index.html" class="main-form__comments form">
+                <fieldset>
+                    <legend>Комментарий</legend>
+                    <textarea name="comment" id="comment" type="text" placeholder="Расскажите обо всех повадках кота"> </textarea>
```

Comment:
id у тебя тут есть 
а вот label с for нету
зачем тогда id? 
добавь скрытый label
```suggestion
<label class="visually-hidden" for="comment">Повадки.</label>              
```
плюс у тексареа нету type